setup.cfg

```diff
@@ -3,7 +3,7 @@ name = wait_for
 author = Peter Savage
 author_email = psavage@redhat.com
 summary = A waiting based utility with decorator and logger support
-description-file = README.rst
+description_file = README.rst
```

Comment:
This is an interesting catch. From what I can tell `description-file` isn't even documented, but both are still read as legacy/compatibility.  It looks like `description-file` works as-is, and the documented way is to use `long_description = file: README.rst`

Reply:
Done